Extra whitespace is being shown when the timestamp is disabled

Can't reproduce.
If you are referring to this case for example:
```
[DEBUG - 2015-07-10T14:43:02.431Z] debug verde
[INFO  - 2015-07-10T14:43:02.432Z] info
[WARN  - 2015-07-10T14:43:02.433Z] warning blink
[ERROR - 2015-07-10T14:43:02.433Z] error
log
[INFO ] info
[WARN ] warning blink
[ERROR] error
```
Where INFO and WARN get an extra space even without timestamp, that's on purpose, so those level block in the beginning of the line are all aligned.

--- a/README.md
+++ b/README.md
@@ -122,6 +122,11 @@ console.enableTimestamp();              //< default enabled
 console.disableTimestamp();
 console.isTimestamped();
 
+// Enable/Disable Equal Spacing on Level Name
+console.enableEqualSpacing();           //< default enabled
+console.disableEqualSpacing();
+console.isEqualSpaced();
+
 // Set OnOutput Callback (useful to write to file or something)
 // Callback: `function(formattedMessage, levelName)`
 console.onOutput(callback);             //< default `null`

--- a/console++.js
+++ b/console++.js
@@ -85,7 +85,8 @@ var _ANSICODES = {
     _messageColored = false,
     _printLevel = true,
     _timed = true,
-    _onOutput = null;
+    _onOutput = null,
+    _spacing = true;
 
 /**
  * Take a string and apply console ANSI colors for expressions "#color{msg}"
@@ -139,7 +140,11 @@ var _decorateArgs = function(argsArray, level) {
         levelMsg;
 
     if (console.isColored()) {
-        levelMsg = _applyColors("#" + console.getLevelColor(level) + "{" + console.getLevelName(level) + "}");
+        levelMsg = console.getLevelName(level);
+        if (!console.isEqualSpaced()) {
+            levelMsg = levelMsg.trim();
+        }
+        levelMsg = _applyColors("#" + console.getLevelColor(level) + "{" + levelMsg + "}");
         msg = _applyColors(msg);
 
         if (console.isMessageColored()) {
@@ -267,6 +272,17 @@ console.disableTimestamp = function() {
 };
 console.isTimestamped = function() {
     return _timed;
+};
+
+// Enable/Disable Equal Spacing on Level Name
+console.enableEqualSpacing = function() {
+    _spacing = true;
+};
+console.disableEqualSpacing = function() {
+    _spacing = false;
+};
+console.isEqualSpaced = function() {
+    return _spacing;
 };
 
 // Set OnOutput Callback (useful to write to file or something)

--- a/test/basic_test.js
+++ b/test/basic_test.js
@@ -43,6 +43,13 @@ console.enableMessageColor();
 console.setLevel(console.LEVELS.ERROR);
 printLevels();
 
-for (i in console) {
+console.setLevel(console.LEVELS.DEBUG);
+console.disableEqualSpacing();
+printLevels();
+
+console.disableTimestamp();
+printLevels();
+
+for (var i in console) {
     console.log(i + " " + typeof(console[i]));
 }

